client/components/LearnerViewDirective.js

```diff
@@ -96,12 +96,24 @@ tie.directive('learnerView', [function() {
               <h3 class="tie-question-title">{{title}}</h3>
               <div class="tie-previous-instructions">
                 <div ng-repeat="previousInstruction in previousInstructions track by $index">
-                  <p ng-repeat="paragraph in previousInstruction track by $index">{{paragraph}}</p>
+                  <div ng-repeat="instruction in previousInstruction track by $index">
+                    <p ng-if="instruction.type == 'text'">{{instruction.content}}</p>
+                    <div ng-if="instruction.type == 'example'">
+                      Example: <pre class="tie-question-example">{{instruction.content}}</pre>
+                    </div>  
+                    <pre class="tie-question-code" ng-if="instruction.type == 'code'">{{instruction.content}}</pre>
```

Comment:
Ignore my previous (not deleted) comment about 'example' type. Is there a reason we need an 'example' type? It appears it just appends 'Example:' prior to the preformatted. If that's the case, why not just have the content author write 'Example:' in the prior text type?